_sass/_components/layout.scss

```diff
@@ -48,11 +48,51 @@
     color: $color-white;
   }
 
-  a {
+  a
+   {
     color: $color-bright;
 
     &:hover {
       color: $color-bright-hover;
     }
   }
+  .usa-button,
+  .usa-button-primary,
+  .usa-button:visited,
+  .usa-button-primary:visited,
+  button,
```

Comment:
Would this change once we change the imports so we can override the USWDS variables?